src/vm/docs/mod.rs

```diff
@@ -264,6 +264,22 @@ Note: This function is only available starting with Stacks 2.1.",
 "#,
 };
 
+const PARSE_PRINCIPAL_API: SimpleFunctionAPI = SimpleFunctionAPI {
+    name: None,
+    signature: "(parse-principal property_name principal)",
+    description: "Note: This function is only available starting with Stacks 2.1.",
+    example: r#"
+"#,
+};
+
+const ASSEMBLE_PRINCIPAL_API: SimpleFunctionAPI = SimpleFunctionAPI {
+    name: None,
+    signature: "(parse-principal property_name principal)",
```

Comment:
change signature here 

also could add examples for both PARSE_PRINCIPAL_API and ASSEMBLE_PRINCIPAL_API?

Reply:
yes.. good points.. what do you think now?

Reply:
looks good to me now!